### PR TITLE
Feature2

### PR DESCRIPTION
**Pull request by aktarshahnazqa**

### CODE REVIEW

**Review by QA-ArsalanAsad — COMMENTED** [review]
All good PO

**Review by MuratOzturk18 — APPROVED** [review]
These are empty but for the time it is okay.

File: hello.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello, It's Shahnaz Aktar
```
Comment (jnfernQA) [inline hello.txt:1]: Looks fine and dandy

**Review by jnfernQA — APPROVED** [review]
Good looks

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by MuratOzturk18 — APPROVED** [review]

**Review by QA-ArsalanAsad — APPROVED** [review]

**Comment by QA-ArsalanAsad** [comment]
finished

**Pull request merged**